src/components/tree/tree.component.js

```diff
@@ -96,6 +97,15 @@ function Tree() {
         showAnimation();
     }
 
+    const findStuff = () => {
+        nodeList = [];
+        if(treeNode.findingKey(treeNode.root, calculateTraversal, parseInt(findKey))){
+          showAnimation();
+        }else{
+          // Key not present handling
+        }
```

Comment:
We can remove this else statement if there is nothing in it.

Reply:
okay will do so.